README.md

```diff
@@ -47,3 +47,9 @@ neuen CSV-Datei in target ausgegeben.
 Hiermit wird, mithilfe der Koordinaten, eine Weltkarte, inklusive markierter Strandorte, erstellt.
 
 Das shellscript führt anschließend Inkscape aus, um den relevanten Bereich vergrößert zu exportieren.
+
+#### for docker 
+#### build image
+docker build -t automate_image -f collaborators-worldmap/docker/Dockerfile .
```

Comment:
The command will fail if the image already exists.. provide a command to make sure that the container can be created or that the previous image is deleted beforehand.